DuckDuckGo/Bookmarks/View/BookmarkListViewController.swift

```diff
@@ -744,6 +753,12 @@ extension BookmarkListViewController: NSSearchFieldDelegate {
     }
 
     private func showSearch(forSearchQuery searchQuery: String) {
+        /// Before searching for the first letter we store the current outline scroll position.
+        /// This is needed because we want to maintain the scroll position in case the search is cancelled.
+        if searchQuery.count == 1 {
```

Comment:
@mallexxx Do you know another alternative where I can save the `lastOutlineScrollPosition` before the search starts? I tried overriding `controlTextDidBeginEditing` and `becomeFirstResponded` but they did not work.